README.md

```diff
@@ -198,7 +198,13 @@ IPU ES2K target.
    git checkout ipdk_v23.07
    ```
 
-3. Build P4-K8s binaries and container images.
+3. Update `inframanager/config.yaml` file with the arp-mac of the ARP proxy
+   interface's MAC. This should be the interface which user intends to select
+   in the step 3 of the Deploy P4 Kubernetes section. Any other changes to
```

Comment:
We should perhaps put link to Deploy P4 Kubernetes section.

Reply:
done